ldes-server-port-ingest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/MemberIngesterImplTest.java

```diff
@@ -59,20 +60,18 @@ void whenValidatorThrowsAnException_thenTheIngestIsAborted_andTheExceptionIsThro
 
 	@Test
 	@DisplayName("Adding Member when there is a member with the same id that already exists")
-	void when_TheMemberAlreadyExists_thenMemberIsReturned() throws IOException {
+	void when_TheMemberAlreadyExists_thenFalseIsReturned() throws IOException {
```

Comment:
Could you change the name of this test please, as we booleans are not used here anymore